ChaosMod/Effects/db/Vehs/VehsBusRain.cpp

```diff
@@ -0,0 +1,34 @@
+#include <stdafx.h>
+/*
+
+	Effect by ProfessorBiddle, a very shite programmer
+
+*/
+static void OnTick()
+{
+
+	Vector3 playerPos = GET_ENTITY_COORDS(PLAYER_PED_ID(), false);
+
+	static DWORD64 lastTick = 0;
+	DWORD64 curTick = GetTickCount64();
+
+	if (curTick > lastTick + 400)

```

Comment:
Every 400 ms seems a bit excessive, doesn’t it?

Reply:
Maybe, it's quite fun to watch and only for 30 seconds. Meteors is every 200ms.

Reply:
> Maybe, it's quite fun to watch and only for 30 seconds. Meteors is every 200ms.

Every 500ms would feel the same and would be plenty as you would be spawning 60 buses over the course of 30 seconds.

Reply:
Fixed